### ISSUE TITLE
FoD: `fcli fod scan start-mobile` fails if `entitlement-id` specified

### ISSUE THREAD

**Issue by kadraman** [fortify/fcli#286]
The `fcli fod scan start-sast` command only works if `entitlement-preference` and `assessment-type` is specified. If `entitlement-id` is specified then an error is returned:

```
java.lang.RuntimeException: Error starting scan:
Request: POST https://api.emea.fortify.com/api/v3/releases/117953/mobile-scans/start-scan?startDate=04/14/2023+15:18&assessmentTypeId&frameworkType=Android&timeZone=UTC&entitlementFrequencyType&entitlementId=2772&fragNo=0&offset=0:
Response: 422 Unprocessable Entity
Response Body:
{"errors":[{"errorCode":null,"message":"The field Int32 is invalid."},{"errorCode":null,"message":"The field EntitlementFrequencyTypes is invalid."}]}
        at com.fortify.cli.fod.rest.helper.FoDFileTransferBase.upload(FoDFileTransferBase.java:152)
        at com.fortify.cli.fod.entity.scan_mobile.helper.FoDMobileScanHelper.startScan(FoDMobileScanHelper.java:78)
        at com.fortify.cli.fod.entity.scan_mobile.cli.cmd.FoDMobileScanStartCommand.getJsonNode(FoDMobileScanStartCommand.java:150)
        at com.fortify.cli.fod.output.cli.AbstractFoDJsonNodeOutputCommand.getJsonNode(AbstractFoDJsonNodeOutputCommand.java:11)
        at com.fortify.cli.common.output.cli.cmd.AbstractOutputCommand.run(AbstractOutputCommand.java:45)
        at picocli.CommandLine.executeUserObject(CommandLine.java:2104)
        at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2539)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2531)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2493)
        at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2351)
        at picocli.CommandLine$RunLast.execute(CommandLine.java:2495)
        at picocli.CommandLine.execute(CommandLine.java:2248)
        at com.fortify.cli.app.FortifyCLI.execute(FortifyCLI.java:86)
        at com.fortify.cli.app.FortifyCLI.main(FortifyCLI.java:68)
```

The options list contains values which are not specific to mobile scans and should be limited. For example:

  - `entitlement-preference`  can only be `SingleScan` or `Subscription`
  - `assessment-type` should only be `Mobile` or `Mobile Plus`

Terminology is also not consistent `entitlementFrequencyType` or `entitlementPreferenceType`?

**Comment by rsenden** [comment]
Fixed by [74121fa](https://github.com/kadraman/fcli/commit/74121faa220d3882f5d102a5155b68b72238e5cc)

**Comment by kadraman** [comment]
Implemented in above commit to develop branch.
